_authors/arbel.md

```diff
@@ -1,3 +1,6 @@
 ---
 title: "Keren Arbel"
 ---
+Keren has been immersed in Buddhist practice since 1997, practicing with variety of teachers in Asia and the West. Keren also holds a PhD in Buddhist Studies.
```

Comment:
Just using the first name is a bit informal. Better to use the full name the first time and "she" after.